moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1937,22 +1944,21 @@ double RobotState::computeCartesianPath(const JointModelGroup* group, std::vecto
     pose.translation() = percentage * rotated_target.translation() + (1 - percentage) * start_pose.translation();
 
     if (setFromIK(group, pose, link->getName(), 1, 0.0, validCallback, options))
-    {
       traj.push_back(RobotStatePtr(new RobotState(*this)));
-    }
```

Comment:
i don't see the point in removing these - they will likely be needed again someday for something

Reply:
consistency with `else`

Reply:
consistency with the else statement